docs/ai/ai-extensions.md

```diff
@@ -0,0 +1,100 @@
+---
+title:  Unified AI building blocks for .NET
+description: Learn how to developer with unified AI building blocks for .NET using Microsoft.Extensions.AI and Microsoft.Extensions.AI.Abstractions libraries
+ms.date: 11/04/2024
+ms.topic: quickstart
+ms.custom: devx-track-dotnet, devx-track-dotnet-ai
+author: alexwolfmsft
+ms.author: alexwolf
+---
+
+# Unified AI building blocks for .NET using Microsoft.Extensions.AI
+
+The .NET ecosystem provides abstractions for integrating AI services into .NET applications and libraries using the [`Microsoft.Extensions.AI`](https://www.nuget.org/packages/Microsoft.Extensions.AI) and [`Microsoft.Extensions.AI.Abstractions`](https://www.nuget.org/packages/Microsoft.Extensions.AI.Abstractions) libraries. The .NET team also enhanced the core `Microsoft.Extensions.*` libraries with these abstractions for .NET Generative AI applications and libraries. In the sections ahead, you learn:
+
+- Core concepts and capabilities of the `Microsoft.Extensions.AI` library.
+- How to work with AI abstractions in your apps and the benefits they offer.
+- Essential AI middleware concepts.
+
+## What is the Microsoft.Extensions.AI library?
+
+`Microsoft.Extensions.AI` is a set of core .NET libraries created in collaboration with developers across the .NET ecosystem, including Semantic Kernel. These libraries provide a unified layer of C# abstractions for interacting with AI services, such as small and large language models (SLMs and LLMs), embeddings, and middleware.
```

Comment:
The package is referred to as a library on line 15 and then a set of libraries here. This might be confusing to readers.